downstream/modules/eda/proc-eda-set-up-rulebook-activation.adoc

```diff
@@ -18,8 +18,16 @@
 +
 Name:: Insert the name.
 Description:: This field is optional.
-Project:: Projects are a logical collection of rulebooks.
-Rulebook:: Rulebooks are shown according to the project selected.
+Organization:: This field is optional.
+Project:: Projects are a logical collection of rulebooks. This field is optional.
+Rulebook:: Rulebooks are displayed according to the project selected.
+Credential:: Select 0 or more credentials for this rulebook activation. This field is optional.
++
+[NOTE]
+====
+The credentials that display in this field are customized based on your rulebook activation and will only include the following credential types: Vault, {PlatformName}, or any custom credential types that you have created. For more information on credentials, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html/event-driven_ansible_controller_user_guide/eda-credentials#eda-credentials[Setting up credentials for {EDAcontroller}].
+====
+//[J. Self] Might need to update the link above for the updated Credentials section.
```

Comment:
If/when updating the link, I'd recommend using the {BaseURL} and {PlatformVers} attributes - might save you some hassle down the line :)